Parquet.Tests/ParquetFileTests.cs

```diff
@@ -120,12 +121,14 @@ public void MergingFilesTest()
 
                 Assert.That(columns1.First().Data.GetValue(0), Is.EqualTo("ResultName"));
                 Assert.That(columns2.First().Data.GetValue(0), Is.EqualTo("ResultName2"));
-                Assert.That(columns1.Skip(1).First().Data.GetValue(0)?.ToString(), Is.EqualTo(stepGuid1.ToString()));
-                Assert.That(columns2.Skip(1).First().Data.GetValue(0)?.ToString(), Is.EqualTo(stepGuid2.ToString()));
+                Assert.That(columns1.Skip(1).First().Data.GetValue(0)?.ToString(), Is.EqualTo(runGuid1.ToString()));
+                Assert.That(columns2.Skip(1).First().Data.GetValue(0)?.ToString(), Is.EqualTo(runGuid2.ToString()));
                 Assert.That(columns1.Skip(2).First().Data.GetValue(0)?.ToString(), Is.EqualTo(parentGuid1.ToString()));
                 Assert.That(columns2.Skip(2).First().Data.GetValue(0)?.ToString(), Is.EqualTo(parentGuid2.ToString()));
                 Assert.That(columns1.Skip(3).First().Data.GetValue(0)?.ToString(), Is.EqualTo(null));
-                Assert.That(columns2.Skip(3).First().Data.GetValue(0)?.ToString(), Is.EqualTo("Test"));
+                Assert.That(columns2.Skip(3).First().Data.GetValue(0)?.ToString(), Is.EqualTo(stepGuid.ToString()));
```

Comment:
We have a test here that the step is actually added. But we can try doing it in a whole plan run too, if that is what you are talking about.